### PR TITLE
Address issue #196

### PR DESCRIPTION
**Pull request by stephanenicolas**
Addresses #196

### CODE REVIEW

File: smoothie/src/main/java/toothpick/smoothie/annotations/ContextSingleton.java [hunk 1]
```diff
@@ -15,6 +16,6 @@
  */
 @Scope
 @Documented
-@Retention(CLASS)
```
Comment (dlemures) [inline smoothie/src/main/java/toothpick/smoothie/annotations/ContextSingleton.java]: can't we remove this class?

Comment (dlemures) [inline smoothie/src/main/java/toothpick/smoothie/annotations/ContextSingleton.java]: or deprecate and remove in TP 2

Comment (stephanenicolas) [inline smoothie/src/main/java/toothpick/smoothie/annotations/ContextSingleton.java]: I will deprecate it for now.

**Comment by dlemures** [comment]
RTM

**Pull request merged**